openedx/core/djangoapps/schedules/tests/test_content_highlights.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
+from openedx.core.djangoapps.schedules.config import COURSE_UPDATE_WAFFLE_FLAG
+from openedx.core.djangoapps.schedules.content_highlights import get_week_highlights, course_has_highlights
+from openedx.core.djangoapps.schedules.exceptions import CourseUpdateDoesNotExist
+from openedx.core.djangolib.testing.utils import skip_unless_lms
+from openedx.core.djangoapps.waffle_utils.testutils import override_waffle_flag
+
+from xmodule.modulestore.tests.django_utils import TEST_DATA_SPLIT_MODULESTORE, ModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory, ItemFactory
+from student.tests.factories import UserFactory
+from student.models import CourseEnrollment
+
+
+@skip_unless_lms
+class TestContentHighlights(ModuleStoreTestCase):
```

Comment:
Super nitpicky, so feel free to blow it off. What do you think of the name "TestOfContentHighlights"?

Reply:
chose to keep as is to remain consistent with other Test class names.